crates/holochain/src/conductor/api/api_external/signal_subscription.rs

```diff
@@ -0,0 +1,69 @@
+use holochain_serialized_bytes::prelude::*;
+use holochain_types::{app::AppId, cell::CellId};
+use std::collections::HashMap;
+
+/// Declares updated Signal subscription settings for an App.
+/// This message is part of the AppInterfaceApi
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize, SerializedBytes)]
+#[cfg_attr(test, derive(PartialEq))]
+pub struct SignalSubscription {
+    /// The app for which to manage subscription
+    app_id: AppId,
+    /// Fine-grained per-cell filters
+    filters: SignalFilterSet,
+}
```

Comment:
I wrote this thinking we'd implement signal subscriptions at this time, but we didn't. I could pull this out but figured I'd leave it in, figuring we'll eventually do something like this.